pkg/gds/emails/client.go

```diff
@@ -49,14 +51,19 @@ func New(conf config.EmailConfig) (m *EmailManager, err error) {
 // EmailManager allows the server to send rich emails using the SendGrid service.
 type EmailManager struct {
 	conf         config.EmailConfig
-	client       *sendgrid.Client
+	Client       EmailClient
```

Comment:
The reason you've made this property public (capitalized) is so that the tests can access the client in order to make assertions? If so, it would be my preference to keep this private and then have a getter method at the bottom of the file: 

```go
// Client returns either the mock or the sendgrid client. This method is primarily used for testing. 
func (m *EmailManager) Client() EmailClient { 
    return m.client 
}
``` 

This ensures that the `EmailClient` can't be modified externally and is a pattern I've adopted in other places, e.g. with `Routes()` on the admin api server.